anx/provider/loadbalancer/address/prefix.go

```diff
@@ -39,6 +42,32 @@ func newPrefix(ctx context.Context, client anxprefix.API, identifier string) (*p
 	return &ret, nil
 }
 
+func newPrefixWithAddressAutodiscovery(ctx context.Context, apiclient api.API, ipamClient ipam.API, identifier string, autodiscoveryTag *string) (*prefix, error) {
```

Comment:
At some point I have to hand over all this information...

Reply:
yep, don't care for those comments too much - if there is an easy way to fix one, fix it - otherwise keep it that way, I'll annoy you if you could fix it ^^